R/cache.R

```diff
@@ -0,0 +1,7 @@
+#' a function that determines the appropriate directory to cache a token
+#' @export
+gptstudio_cache_directory = function(){
+  rappdirs::user_data_dir(appname = glue::glue("gptstudio"),
```

Comment:
can we switch this to `tools::R_user_dir()`?

Reply:
Agreed. Will do.